VTUL/dlp-access
# Reload sections of collection show page without refresh

**Reload sections of collection show page without refresh.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2320) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Reload sections of collection show page without refresh

# What's the changes? (:star:)
* Move collection query into CollectionShowPage and remove CollectionShowLoader so query can be called on `componentDidUpdate` 
* change `<a>` tags to `NavLink` to prevent browser refresh
* Fix breadcrumbs at the top to respond to route changes without browser refresh
* Refactor CollectionShow to have "top content" in separate component to try to get it to remain static when staying within the same top-level collection
* Fix ScrollToTop to work without browser refresh
* Refactor all sub-components of CollectionShowPage to respond on `componentDidUpdate`

# How should this be tested?

A description of what steps someone could take to:
* Visit a collection and click on the various sub-collections. You should see appropriate sections updating without browser refresh.

# Additional Notes:
* branch: `LIBTD-2320`

# Interested parties
@yinlinchen 

(:star:) Required fields
